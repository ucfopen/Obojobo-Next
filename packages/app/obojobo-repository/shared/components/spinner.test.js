import React from 'react'
import Spinner from './spinner'
import { create, act } from 'react-test-renderer'

describe('Spinner', () => {
	test('Spinner component', async () => {
		let component
		await act(async () => {
			component = create(<Spinner />)
		})

		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
