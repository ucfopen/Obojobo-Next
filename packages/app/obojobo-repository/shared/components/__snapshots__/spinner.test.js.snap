// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Spinner component 1`] = `
<div
  aria-label="Loading content"
  className="obojobo-draft--components--spinner"
>
  <div
    aria-hidden={true}
    className="spinner"
  >
    <svg
      preserveAspectRatio="xMidYMid"
      viewBox="0 0 100 100"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="50"
        cy="50"
        fill="none"
        r="35"
        stroke="black"
        strokeDasharray="60 110"
        strokeWidth="10"
        transform="rotate(0.82332 50 50)"
      />
    </svg>
  </div>
</div>
`;
